charts/hlf-k8s/README.md

```diff
@@ -101,7 +106,8 @@ The following table lists the configurable parameters of the hlf-k8s chart and d
 | `privateCa.enabled` | if true, use a private CA | `false` |
 | `privateCa.configMap.name` | The name of the ConfigMap containing the private CA certificate | `private-ca` |
 | `privateCa.configMap.fileName` | The CA certificate filename within the ConfigMap | `private-ca.crt` |
-| `hooks.deleteSecrets.enabled` | If true, the secrets created by the chart will be automatically deleted when the chart is uninstalled | `true` |
+| `hooks.deleteSecrets.enabled` | If true, the HLF crypto materials secrets created by the chart will be automatically deleted when the chart is uninstalled | `true` |
+| `hooks.deleteCCIDSecrets.enabled` | If true, the chaincode ccid secrets created by the chart will be automatically deleted when the chart is uninstalled | `true` |
```

Comment:
why create an additionnal flag? I would have used deleteSecrets for simplicity here

Reply:
Because, you may want to keep hlf-secrets (crypto materials) but not chaincode CCID which is directly link to the installation of the chaincode